godtools/XMLContent/KGP/kgp-5.xml

```diff
@@ -100,7 +99,7 @@
                 </content:text>
             </paragraph>
             <paragraph>
-                <content:button type="event" value="kgp-6">
+                <content:button type="event" value="kgp-6,send-information-modal">
```

Comment:
Yeah, what I'm doing is to have a boolean which will detect if opening or closing, but, instead of that, what should we do to determine if closing or opening? Maybe sending parameters? like `<content:button type="event" value="kgp-6,send-information-modal:close">`?